api/main.go

```diff
@@ -75,9 +75,10 @@ func main() {
 	api.Use(middleware.InstanceJWT().MiddlewareFunc())
 	{
 
-		filters := api.Group("/filters/:report/:section/:view")
+		filters := api.Group("/filters")
 		{
 			filters.GET("", methods.GetDefaultFilter)
```

Comment:
Maybe this endpoint is useless 🤔 . I think the below one is enough to handle both empty or defined `:filter`

Reply:
Did a couple of tests, looks like both of them are needed for the optional parameter to work properly